index.js

```diff
@@ -0,0 +1,11 @@
+function openNav() {
+  document.getElementById('mobile-menu').classList.add('visible');
+}
+
+function closeNav() {
+  document.getElementById('mobile-menu').classList.replace('visible', 'hidden');
+}
```

Comment:
- [ ] good job on the menu, however, the close button should be an X as shown in the template, and the menu should cover the entire screen including the navbar.